Angular/src/app/components/filter-form/config.ts

```diff
@@ -0,0 +1,15 @@
+export interface IFilterInputOptions {
+  placeholder: string
+  onKeyUp: any
```

Comment:
Try to avoid 'any' type, because it's a little dangerous and breaks the type's system.

In this case, you can try something like this:
```ts
export interface OnKeyUpEvent {
  event: { currentTarget: { value: string}}
}

export interface IFilterInputOptions {
  value: string;
  placeholder: string
  onKeyUp: (event: OnKeyUpEvent) => void;
}
```